apps/sms/tools/update_nga_libs.sh

```diff
@@ -27,6 +32,10 @@ fi
 
 bower install $threads_repo $sww_repo
 
-browserify $threads_index --standalone $threads_name > $threads_path
+# Threads lib preparing.
+[ -d $threads_destination ] || mkdir $threads_destination
+browserify $threads_index --standalone $threads_alias > $threads_index_destination
+cp -r $threads_plugins $threads_destination
```

Comment:
try to use double quotes between your variables holding file names -- not sure this is really useful here but it's a good thing to take the habit :)